src/Components/Generator/ActionButton.js

```diff
@@ -5,7 +5,8 @@ import MailtoDropdown from '../MailtoDropdown';
 
 export default class ActionButton extends Component {
     render(props) {
-        const enabled = props.transport_medium === 'email' ? !!props.letter : props.download_active;
+        const enabled =
+            props.transport_medium === 'email' ? !!props.letter && props.ready : props.download_active && props.ready;
```

Comment:
I personally would prefer this but that's not too important.

```suggestion
        const enabled = (props.transport_medium === 'email' ? !!props.letter : props.download_active) && props.ready;
```